Products/ZenModel/migrate/addZeneventserverstdioLogFilters.py

```diff
@@ -0,0 +1,55 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2017, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+import Migrate
+import os
+import servicemigration as sm
+from Products.ZenUtils.Utils import zenPath
+from Products.ZenModel.ZMigrateVersion import SCHEMA_MAJOR, SCHEMA_MINOR, SCHEMA_REVISION
+
+log = logging.getLogger("zen.migrate")
+sm.require("1.1.10")
+
+class AddZeneventserverstdioLogFilters(Migrate.Step):
+    """
+    Add LogFilters for zeneventserver-stdio logs issue addressed by ZEN-28081
+    """
+    version = Migrate.Version(SCHEMA_MAJOR, SCHEMA_MINOR, SCHEMA_REVISION)
+
+    def cutover(self, dmd):
+        try:
+            ctx = sm.ServiceContext()
+        except sm.ServiceMigrationError:
+            log.info("Couldn't generate service context, skipping.")
+            return
+
+        service = ctx.getTopService()
+        log.info("Found top level service: '{0}'".format(service.name))
+        if service.name.find("Zenoss") != 0 and service.name.find("UCS-PM") != 0:
+            log.info("Top level service name isn't Zenoss or UCS-PM; skipping.")
+            return
+
+        filterName = "zeneventserver-stdio"
```

Comment:
Unless I missed something, the migration is incomplete because the `Filters` property is not set for the `LogConfig` entry for the zeneventserver-stdio log file in the zeneventserver service.   Another reason to consider combining this migration logic with the filepath change into a single script